index.html

```diff
@@ -981,9 +981,16 @@ <h2><dfn>Coalesced events</dfn></h2>
             <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events list</dfn> (a list of
             zero or more <code>PointerEvent</code>s). For trusted <code>pointermove</code> and
             <code>pointerrawupdate</code> events, the list is a sequence of all <code>PointerEvent</code>s
-            that were coalesced into this event. For all other trusted event types, it is an empty list.
-            Untrusted events have their <a>coalesced events list</a> initialized to the value passed to the
-            constructor.</p>
+            that were coalesced into this event. The "parent" trusted <code>pointermove</code> and
+            <code>pointerrawupdate</code> event represents an accumulation of these coalesced events,
+            but may have additional processing (for example to align with the display refresh rate).
+            For all other trusted event types, it is an empty list. Untrusted events have their
+            <a>coalesced events list</a> initialized to the value passed to the constructor.</p>
```

Comment:
I'm not sure it is clear from this that the coalesced events list contains always at least one event.

Reply:
maybe to be overly explicit then, after `(for example to align with the display refresh rate).`, cram in an extra sentence.

```
(for example to align with the display refresh rate). As a result, the coalesced events lists always contains one or more events. For all other trusted event types, ...
```

Reply:
`one or more events` could also be changed to `at least one event`. question of taste i guess

Reply:
Thanks @flackr ... while a bit awkward ("events events events"), I think that's now quite clear

Reply:
Suggesting one minor tweak: replace

> As a result, the coalesced events lists for these events always contain at least one event

with

> The coalesced events lists for these two event types always contain at least entry

Then the following sentence about "for all _other_ ..." becomes a bit clearer to me.  Wdyt?